client/styles.css

```diff
@@ -0,0 +1,87 @@
+body{
+    background-color: rgb(24, 32, 32);
+}
+
+form{
+    display: flex;
+    flex-flow: row;
+    justify-content: center;
+    font-family: Inconsolata, 'times new roman';
+}
+
+.flex-container{
+    align-items: center;
+    justify-content: center;
+    height: 90vh;
+}
+#inputs {
+    padding: 15px;
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: space-around;
+    align-items: flex-start;
+    width: 45vw;
+}
+
+#button{
+    display: flex;
+    align-items: center;
+    justify-content: left;
+    width: 20vw;
+}
+
+input { 
+    background-color: rgb(24, 32, 32);
+    color: green;
+    width: 40vw;
+    padding: 10px;
+    margin: 20px;
+    font-size: 30px;
+    border: none;
+    font-family: Inconsolata, 'times new roman';
+}
+
+input:hover{
+    border: 2px solid green;
+    box-shadow: 0 5px 30px 0 rgba(55, 219, 68, 0.2);
+}
+
+#title{
+    font-size: 45px;
+}
+::placeholder{
+    color: green;
+}
+
+button {
+    padding: 10px;
+    margin: 10px;
+    color: green;
+    background-color: rgb(36, 48, 48);
+    font-size: 30px;
+    border-radius: 10%;
+    transition-duration: 0.5s;
+    border: 3px rgb(42, 58, 45) solid;
+    
+}
+
+button:hover{
+    background-color: green;
+    color:silver;
+    /* box-shadow: 0 12px 16px 0 rgba(55, 219, 68, 0.24), 0 17px 50px 0 rgba(55, 219, 68, 0.19); */
+    box-shadow: 0 10px 40px 0 rgba(55, 219, 68, 0.4);
+}
+
+
+h1{
```

Comment:
these probably got forgotten! do not forget about your css, she deserves love and attention I promise